frontend/src/components/common/FormikDatePicker.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { FunctionComponent } from 'react';
```

Comment:
@Fosol did you intend to include this file in your PR? 

@devinleighsmith - didn't you renamed/moved this file?

Reply:
I think this was a merger mistake.  I’ll double check

Reply:
Yes, this looks like the old version.